maps.asm

```diff
@@ -1219,3 +1219,8 @@ INCLUDE "data/maps/headers/AgathasRoom.asm"
 INCLUDE "scripts/AgathasRoom.asm"
 INCLUDE "data/maps/objects/AgathasRoom.asm"
 AgathasRoom_Blocks: INCBIN "maps/AgathasRoom.blk"
+
+
+SECTION "Hide/Show Data", ROMX
```

Comment:
Please put this section at the start of the file, since it's different than all the rest and also appeared before "Maps 1" in main.asm anyway.